app/main.py

```diff
@@ -16,4 +16,16 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+values = [lucky_number, pi, one_is_a_prime_number]
+values += [name, my_favourite_films, profile_info, marks, collection_of_coins]
+sorted_variables = {"mutable": [], "immutable": []}
+
+mutable_values = (list, dict, set)
+immutable_values = (int, str, tuple, type(None), float, bool)
+
+for el in values:
+    if isinstance(el, mutable_values):
+        sorted_variables["mutable"].append(el)
+
+    elif isinstance(el, immutable_values):
+        sorted_variables["immutable"].append(el)
```

Comment:
In this task, you just need to fill the `sorted_variables` dictionary with the required values, no additional logic is required.